tests/download/test_sentinelhub_client.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from sentinelhub import SentinelHubDownloadClient, SentinelHubStatisticalDownloadClient, SHConfig
+
+
+@pytest.fixture(name="blank_config")
+def blank_config_fixture():
+    config = SHConfig()
+    config.reset()
+    return config
```

Comment:
is this `SHConfig(use_defaults=True)`?

Reply:
You are right, I have to get used to new functionalities :sweat_smile: